spec/reek/cli/xml_report_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../spec_helper'
+require_relative '../../../lib/reek/examiner'
+require_relative '../../../lib/reek/cli/report/report'
+require_relative '../../../lib/reek/cli/report/formatter'
+
+describe Reek::CLI::Report::XMLReport do
+  let(:instance) { Reek::CLI::Report::XMLReport.new }
+
+  context 'empty source' do
+    let(:examiner) { Reek::Examiner.new('') }
+
+    before do
+      instance.add_examiner examiner
+    end
+
+    it 'prints empty checkstyle xml' do
+      expect { instance.show }.to output("<?xml version='1.0'?>\n<checkstyle/>\n").to_stdout
+    end
```

Comment:
Can you please add one more spec with a non-empty document so that it's obvious what kind of output is expected?
Just the empty doc above doesnt tell you much.:)